compiler/rustc_target/src/spec/mipsel_unknown_none.rs

```diff
@@ -0,0 +1,41 @@
+// Bare MIPS32r2, little endian, softfloat
+// can be used for MIPS M4K core (e.g. on PIC32MX devices)
```

Comment:
```suggestion
//! Bare MIPS32r2, little endian, softfloat
//!
//! Can be used for MIPS M4K core (e.g. on PIC32MX devices)
```

Can you also add to the comment the ABI that's used by this? I'd expect it to be O32?

Reply:
Well, it is the typical MIPS ABI and calling convention. What is O32?

Reply:
Also updating platform-support.md? The tools are complaining about it.

Reply:
O32 is the oldest and slowest MIPS ABI, but also apparently the most commonly-used one: https://en.wikipedia.org/wiki/MIPS_architecture#Calling_conventions

According to LLVM, it uses O32 for the normal `mipsel` triples.

> Also updating platform-support.md? The tools are complaining about it.

Yes, CI checks that every supported target is in the list here: https://github.com/rust-lang/rust/blob/master/src/doc/rustc/src/platform-support.md